src/main/java/net/greatstart/controllers/UserController.java

```diff
@@ -1,14 +1,20 @@
 package net.greatstart.controllers;
 
 
+import net.greatstart.dto.DtoUser;
 import net.greatstart.dto.DtoUserProfile;
 import net.greatstart.mappers.UserProfileMapper;
 import net.greatstart.model.User;
+import net.greatstart.services.SecurityService;
 import net.greatstart.services.UserService;
+import net.greatstart.validators.UserValidationService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.validation.Errors;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.validation.Errors'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)